tds.go

```diff
@@ -1292,6 +1316,15 @@ initiate_connection:
 				break
 			}
 
+			// Save options returned by the server so callers implementing
+			// proxies can pass them back to the original client.
+			switch tok.(type) {
+			case envChangeStruct, loginAckStruct, doneStruct:
```

Comment:
is `featureExtAck` new? do we need to include it?

Reply:
I think we can skip for now.